driver-core/src/main/com/mongodb/internal/Locks.java

```diff
@@ -54,7 +54,7 @@ public static void withLockAsync(final StampedLock lock, final AsyncRunnable run
         }, callback);
     }
 
-    public static void withLock(final StampedLock lock, final Runnable runnable) {
+    public static void withInterruptibleLock(final StampedLock lock, final Runnable runnable) {
```

Comment:
- If this method is renamed, then its asynchronous counterpart `withLockAsync(StampedLock, AsyncRunnable, SingleResultCallback<Void>)` should also be renamed similarly
- `withLockAsync should also be declared as `throws MongoInterruptedException` consistently with the rest of the `Locks` methods that use interruptible locking.

Reply:
Deleted dead code